templates/pkg/resource/manager.go.tpl

```diff
@@ -186,6 +187,7 @@ func (rm *resourceManager) onError(
 			condition.Status == corev1.ConditionTrue {
 			// resource is in Terminal condition
 			// return Terminal error
+			rm.log.Info("Terminal condition is set to true for resource.")
```

Comment:
remove this please.

Reply:
During debugging, I found this log helpful.

Reply:
Please remove this from the template, Kumar. If we want to keep the original error reference, we should be using `errors.WithMessage()` and returning the wrapped error. The top-level reconciler logs errors so we shouldn't duplicate that logging here.

https://godoc.org/github.com/pkg/errors#WithMessage

For example, we could do the following here:

```go
errMsg := fmt.Sprintf("terminal condition: %s", err)
return r1, errors.WithMessage(ackerr.Terminal, errMsg)
```

Reply:
Removed the changes done to template file.